squarelet_auth/users/utils.py

```diff
@@ -65,7 +67,7 @@ def _squarelet_update_or_create(uuid, data):
         "use_autologin": True,
     }
     user_data = {user_map[k]: data.get(k, user_defaults[k]) for k in user_map}
-    return User.objects.update_or_create(uuid=uuid, defaults=user_data)
```

Comment:
removing this because getting by the uuid won't work if there is an existing admin user in the agenda watch app and that user uses the same email in muckrock.

in other words. i created a superuser with smcalilly@gmail.com, which created a uuid. then tried to authenticate with muckrock with the same email. the uuid in muckrock is different than the original superuser, which resulted in a duplicate error since we're getting by uuid but using the same email/username. so this essentially treats the muckrock uuid as the source of truth and overwrites the uuid of any existing superusers in the agenda watch database.